composer.json

```diff
@@ -91,14 +95,14 @@
         "predis/predis": "^1.1",
         "pusher/pusher-php-server": "~4.0",
         "sixlive/nova-text-copy-field": "^1.5",
-        "skymeyer/vatsimphp": "1.0.*",
+        "skymeyer/vatsimphp": "dev-master",
```

Comment:
You should be able to use `"skymeyer/vatsimphp": "^2.0"` now.
https://github.com/skymeyer/Vatsimphp/blob/master/docs/index.md#installation

Reply:
> You should be able to use `"skymeyer/vatsimphp": "^2.0"` now.
> https://github.com/skymeyer/Vatsimphp/blob/master/docs/index.md#installation

Thanks for your help, I've updated this and it appears to work as expected on `^2.0`